### ISSUE TITLE
Paper, equation (5)

### ISSUE THREAD

**Issue by ndvbd** [xhan77/ssd-lm#15]
Hi! Thanks for the paper! A few questions:

1. In equation 5, K is not defined. Is it just an hyperparam?
2. Why -K and not 0, like in regular one hot? To make the softmax more extreme/discrete?
3. In equation (8) I think you forgot to put the 'sotftmax' as a wrapper for the logits, as the model receives the softmax simplex. (before the softmax, it is not a simplex)
4. In eq (10), why did you write 'logits'? After all, you wrote 'which are converted to a dist using softmax' so it should be probability, not logits
5. in eq (10) do you predict the *original* tokens, or the previous timestep (meaning less noisy tokens)?
6. In table 1, in the PPL, why isn't the arrow pointing down as in 'lower is better'?
7. Re the sentence: "Prior works, however, have shown that low perplexity of generated text is not necessarily an indication of high quality but of degenerate behavior (Nadeem et al., 2020; Zhang et al., 2021) and have proposed closeness to the perplexity of human-written text as a better evaluation. Hence, we also report the difference of log perplexity between the generated text and human-written continuations" - I couldn't find the word 'perplexity' in the cited article. What is the formula that you use? Do you calc the difference in log perplexity of the gold continuation as prediction by GPT NEO to the prediction of GPT NEO of the text generated by the specific model you test?
